endpoints/wellknown.py

```diff
@@ -24,7 +24,10 @@ def app_capabilities():
         "appName": "io.quay",
         "capabilities": {
             "io.quay.view-image": {"url-template": view_image_tmpl,},
-            "io.quay.image-security": {"rest-api-template": image_security_tmpl,},
+            "io.quay.image-security": {
+                "rest-api-template": image_security_tmpl,
+                "deprecated": True,
```

Comment:
@josephschorr Does it make sense to add this here?

Reply:
Yup!